crates/tools/js/benchmark/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "benchmark",
+  "version": "1.0.0",
+  "private": true,
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "benchmark": "node index.js",
+    "prebenchmark": "cd ../../../edr_napi/ && pnpm build && cd ../../packages/hardhat-core/ && pnpm build"
```

Comment:
How will we do this when we move to a separate repo? Should we already follow that process?

cc: @fvictorio

Reply:
It'd need `verdaccio` I think if we treated `hardhat-core` as a separate repo to be able to use custom version of `edr` with it. `verdaccio` means we'd need a Docker runner for consistent results locally, so I'd rather cross that bridge once we get there.

Reply:
My feeling is that this should live on the Hardhat side. Linking EDR on it is something we'll do often, so it's better to always do it in one direction. The problem is how this would play with the automated benchmark runs, but that doesn't seem like an impossible problem to figure out.